src/components/live/sponsors.tsx

```diff
@@ -1,5 +1,126 @@
+import Image from "next/image";
+
+interface Sponsor {
+  name: string;
+  logo: string;
+  link: string;
+}
+
+const SponsorCard = ({ name, logo, link }: Sponsor) => {
+  return (
+    <a
+      className="mx-auto flex h-36 w-full cursor-pointer select-none justify-center rounded-xl bg-primary-700 p-4 align-middle transition-all hover:scale-[1.0075] hover:shadow-2xl xl:h-40 2xl:h-48"
+      href={link}
+      target="_blank"
+      rel="noreferrer noopener"
+    >
+      <Image
+        src={logo}
+        alt={name}
+        width={0}
+        height={0}
+        className="h-auto w-full"
+      />
+    </a>
+  );
+};
+
 const Sponsors = () => {
-  return <div>sponsors</div>;
+  return (
+    <div className="p-12">
+      <h1 className="py-4 font-DM_Sans text-3xl font-bold ">Title Sponsor</h1>
+      <div>
+        <SponsorCard
+          name="Scotiabank"
+          logo="/sponsors/Scotiabank.svg"
+          link="https://www.scotiabank.com/"
+        />
+      </div>
+      {/* Diamond Sponsors */}
+      <h1 className="py-4 font-DM_Sans text-3xl font-bold">Diamond Sponsors</h1>
```

Comment:
```suggestion
      <h1 className="py-4 font-DM_Sans text-2xl font-medium text-slate-600">Diamond Sponsors</h1>
```

Could you change all the headings to this? So that it matches the style from the design

Reply:
yes sir!